src/fluree/db/query/analytical.cljc

```diff
@@ -420,7 +420,11 @@
                     (let [partition (dbproto/-c-prop db :partition (last clause))
                           max-sid   (-> db :ecount (get partition))
                           min-sid   (flake/min-subject-id partition)
-                          flakes    (<? (query-range/index-range db :spot >= [max-sid] <= [min-sid]))
+                          flakes    (if-not max-sid
```

Comment:
This is a nit-pick, but a tiny readability preference I have is to avoid negated conditionals if I can simply swap the then/else clauses. It saves me an extra brain cycle when reading later. But I'll defer to your judgment either way :+1:

Reply:
Yeah, I tend to agree with you, but I explicitly picked `if-not` this time because the deciding factor in my mind was whether or not `max-sid` was null. However, this is completely arbitrary, so I changed it to an `if` because there should be a non-arbitrary reason to use `if-not`. Thanks for the suggestion.